assets/js/components/ChecksCatalog/ChecksCatalog.stories.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { faker } from '@faker-js/faker';
+
+import { catalogCheckFactory } from '@lib/test-utils/factories';
+import ChecksCatalog from './ChecksCatalog';
+
+const groupName1 = faker.string.uuid();
+const groupName2 = faker.string.uuid();
+const groupName3 = faker.string.uuid();
+const group1 = catalogCheckFactory.buildList(5, { group: groupName1 });
+const group2 = catalogCheckFactory.buildList(5, { group: groupName2 });
+const group3 = catalogCheckFactory.buildList(5, { group: groupName3 });
+const catalogData = group1.concat(group2).concat(group3);
```

Comment:
What do you think about using the spread operator as an alternative ? 
`const catalogData1 = [...group1, ...group2, ...group3];`
Just wondering which we should prefer :)

Reply:
I would say in this case it is pretty irrelevant for us, the array elements number is small.
By default, I think that using `concat` is more efficient and meaningful than spread operator for arrays.
PD: Indeed, i have done a [research](https://stackoverflow.com/questions/48865710/spread-operator-vs-array-concat), and it looks like `concat` is faster for elements that we know are arrays.

Reply:
Interesting thanks!